integration/test/plan/non_mpi_profiling/test_multi_app.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#  Copyright (c) 2015 - 2024 Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+mkdir -p ${TMPDIR}
+
+INPUT_FILE=$(mktemp)
+cat > ${INPUT_FILE} << "EOF"
+{
+    "loop-count": 50,
+    "region": ["stream", "dgemm"],
+    "big-o": [3.0, 30.0]
+}
+EOF
+
+export SYSTEMD_BUS_TIMEOUT=600
+
+TEST_NAME=test_multi_app
+export GEOPM_PROFILE=${TEST_NAME}
+export GEOPM_PROGRAM_FILTER=geopmbench,stress-ng
+export LD_PRELOAD=libgeopm.so.2.1.0
+
+# GEOPM_CTL_LOCAL=true \
+GEOPM_REPORT=${TEST_NAME}_report.yaml \
+GEOPM_REPORT_SIGNALS=TIME@package \
+GEOPM_NUM_PROC=2 \
+setsid geopmctl &
+
+# Some of the initialization regions of geopmbench finish quickly. The
+# test_multi_app.py test checks the execution time of all regions. This sleep
+# ensures that geopmctl is started before starting geopmbench.
+sleep 2
```

Comment:
Now that the runtime for each benchmark is longer, maybe the sleep 2 is not needed?